apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.html.ts

```diff
@@ -74,14 +102,27 @@ export function InstructorCourseAdminSettings({
                 </div>
                 <div class="form-group">
                   <label for="display_timezone">Timezone</label>
-                  <input
-                    type="text"
+                  <select
```

Comment:
Nit: maybe move this above the "Institution" part so that we can separate the editable fields from the non-editable ones?

Reply:
Moved the timezone field above the institution field in [1f9d924](https://github.com/PrairieLearn/PrairieLearn/pull/9581/commits/1f9d9242b1ab757397ce2a399f68924f58891b8e).